packages/nimble-components/src/table-column/extensions/tests/fractional-width-column.spec.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable max-classes-per-file */
+import { ViewTemplate, ElementStyles, html } from '@microsoft/fast-element';
+import { fixture, Fixture } from '../../../utilities/tests/fixture';
+import type { TableCellState } from '../../base/types';
+import { TableColumn } from '../../base';
+import {
+    FractionalWidthColumn,
+    fractionalWidthColumn
+} from '../fractional-width-column';
+
+class TestTableColumnBase extends TableColumn {
+    public cellTemplate: ViewTemplate<TableCellState> = html``;
+    public cellStyles?: ElementStyles | undefined;
+    public cellRecordFieldNames: readonly string[] = [];
+}
+
+class TestTableColumnBaseMixin extends TestTableColumnBase {}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface TestTableColumnBaseMixin extends FractionalWidthColumn {}
+
+class TestTableColumn extends fractionalWidthColumn(TestTableColumnBaseMixin) {}
+
+const composedTestTableColumn = TestTableColumn.compose({
+    baseName: 'fractional-width-test-table-column'
```

Comment:
@rajsite this file is where I was getting the intermittent test failures that I was trying to describe earlier.
If you look at `table-column.spec.ts` and this file, you'll see similar test setup (like the `TestTableColumn.compose(...)` calls). Changing `baseName` here to be different than the one used in the other test file seemed to resolve the test issues.

Reply:
This is the pattern fast test use to create temporary dummy elements. Has a bit less boilerplate and is copy-paste safer:

https://github.com/ni/nimble/blob/1fef9860af8a495a31f2fb89e073a91b86d6bfcd/packages/nimble-components/src/utilities/tests/tests/fixture.spec.ts#L18-L30

Reply:
I think we can revisit this at a later time if we feel strongly about it.